Makefile

```diff
@@ -68,6 +68,13 @@ install:
 clean:
 	rm -rf bin/*
 
+## creates a new release tag
+.PHONY: release/tag
+release/tag: VERSION?=v$(shell cat VERSION)
```

Comment:
The tag needs a "v" on the prefix because of how Go handles versions (v<semver>, like the current releases: https://github.com/mineiros-io/terramate/releases).